### ISSUE TITLE
Beta testers get same version as non-beta testers

### ISSUE THREAD

**Issue by jidanni** [commons-app/apps-android-commons#3976]
1. Visit https://play.google.com/store/apps/details?id=fr.free.nrw.commons , note the version number available.
2. Now sign up as a Beta Tester... `You're a beta tester for this app. Awesome!`
3. Now note the version available. It is the same version.

**Comment by nicolas-raoul** [comment]
Yes, the Google Play alpha/beta is pretty weird, it takes time to get the intended version.

The surest way to know is to compare the commit code in About. The latest commit codes can be seen at https://github.com/commons-app/apps-android-commons/commits/master (use the dropdown to select any other branch).

Cheers!

**Comment by jidanni** [comment]
You must mean "master". That's the only active branch other than 2.13.2~757c7b008 .

**Comment by nicolas-raoul** [comment]
Yes these are the most active branches currently on the main repository, you may be interested in seeing work being done in other branches too: https://github.com/commons-app/apps-android-commons/network

**Comment by jidanni** [comment]
Conclusion: there is only
- Master
- Google Play version
- Google Play Beta version
- Fdroid Version.

And
- Google Play version
- Google Play Beta version
are clamped to the same version, for no good reason.

**Comment by jidanni** [comment]
Wait! On https://github.com/commons-app/apps-android-commons/issues/3969#issuecomment-713749200 I finally found the Alpha!!

**Comment by misaochan** [comment]
Currently beta testers get the same version as production users, because the latest beta has been promoted to production and we do not have a release in beta testing. They are not always the same. 

Master and alpha are (supposed to be) the same version and they should almost always be different from beta and production.
